src/components/Pressable/GenericPressable/PropTypes.js

```diff
@@ -4,6 +4,20 @@ import CONST from '../../../CONST';
 
 const stylePropTypeWithFunction = PropTypes.oneOfType([stylePropType, PropTypes.func]);
 
+/**
+ * Marks prop as required if the component is accessible
+ * @param {Object} props
+ * @param {String} propName
+ * @param {String} componentName
+ * @returns {Error} Error if prop is required
+ */
+function requiredIfAccessible(props, propName, componentName) {
+    if (props.accessible !== true || (props[propName] !== undefined && typeof props[propName] === 'string')) {
```

Comment:
@robertKozik

Reply:
@robertKozik  I don't think it would be scalable this way as well. Since we already tied the logic to `props.accessible !== true` and only props that have linkage with `accessible` can use this. And the prop has to be `typeof props[propName] === 'string'` string type as well. It super narraws this function use and only a replica of `accessibilityLabel` or similar could use this. 

So I think `props.accessibilityLabel` seems similar. `But I don't have a strong concern either since there is no drawback to this here as well`
Was the above mentioned considered and your thoughts on the above?

Reply:
Nevermind. I am fine with this since there is no drawback as well and going ahead to test it out in other platforms.

Reply:
It could be used for accessibilityHint as well. I'm not that connected to my approach, so I can change it if you prefer

Reply:
I think we can have it `props.accessibilityLabel` instead of the generic way, similar to the below. This way it will allow to reuse the same function for different prop types by directly accessing the prop. Currently it is limited to only string type.

https://github.com/Expensify/App/blob/5fab2de66ee1666d75ef6bae4f56fd04f94eb6f6/src/components/CheckboxWithLabel.js#L11-L23

Reply:
Could you kindly change this to use `props.accessibilityLabel` directly? I don't think I have any more comments

Reply:
@robertKozik

Reply:
Once done, Kindly resolve this thread and tag me, so that i can continue on the reviewers checklist.

Reply:
@abdulrahuman5196 Done 🚀